tests/target/issue-1681.rs

```diff
@@ -0,0 +1,19 @@
+// rustfmt-max_width: 80
+
+fn foo() {
+    // This is where it gets good
```

Comment:
Could remove these comments - not clear if they relate to the test or are just copied in